pkg/webui/components/footer/index.js

```diff
@@ -51,12 +57,18 @@ const Footer = ({ className, documentationLink, links, supportLink, onlineStatus
       <span className={style.version}>v{process.env.VERSION}</span>
       {documentationLink && (
         <Link.Anchor className={style.documentation} secondary href={documentationLink} external>
-          <Message content={sharedMessages.documentation} />
+          <Message
+            content={
+              isMinimized && supportLink
+                ? sharedMessages.documentationAbbreviation
+                : sharedMessages.documentation
+            }
+          />
         </Link.Anchor>
       )}
       {supportLink && (
         <Button.AnchorLink
-          message={sharedMessages.getSupport}
+          message={isMinimized ? null : sharedMessages.getSupport}
```

Comment:
```suggestion
          message={isMinimized ? undefined : sharedMessages.getSupport}
```

This will make the button use the correct styling for only icon buttons.